debian/changelog

```diff
@@ -1,3 +1,10 @@
+tiqit (1.1.0-1) trusty; urgency=low
+
+  * Move to python3
+  * Remove browser check
```

Comment:
Add a few more words so it's clearer what check this is referring to. (And add full stops to both bullets.)

Reply:
Done